### PR TITLE
Removeversion - removes version as a configurable setting...

### LINKED ISSUE #48: Version field should be renamed "launch version" and used accordingly.
The plugin doesn't in fact support 0.9 - it breaks when you try. Instead, we should use 1.0.2 for all communication between Moodle and the LRS and only use the version setting when building the launch URL.

### PR DESCRIPTION
**Pull request by davidpesce**
...And hardcodes it to v1.0.0

This fixes #48 and #50.

### CODE REVIEW

**Comment by davidpesce** [comment]
Crap, it added the commits from the other PR as well. I haven't done it this way before. When you accept the other PR, will these commits disappear from this PR?

**Comment by garemoko** [comment]
Yeh. The way to avoid this next time is to have your master branch match the repo and create a branch off master for each PR.

**Comment by davidpesce** [comment]
Sounds good. Will do from here on out.

File: lib.php [hunk 1]
```diff
@@ -94,9 +113,43 @@ function tincanlaunch_update_instance(stdClass $tincanlaunch, mod_tincanlaunch_m
...
+            $DB->delete_records('tincanlaunch_lrs', array('id' => $tincanlaunch_lrs_id));
+        }
+    }
+
+    if(!$DB->update_record('tincanlaunch', $tincanlaunch)){
+        return false;
+    }
 
-    return $DB->update_record('tincanlaunch', $tincanlaunch);
+    return true;
```
Comment (garemoko) [inline lib.php:152]: Why this change?

**Comment by garemoko** [comment]
It looks like there's a lot of stuff in this PR not related to the version change, which should be pretty minor. Can you explain, or make the PR with just the version changes?

**Comment by garemoko** [comment]
Just testing this version of the code and it's var-dumping the settings out onto the screen on view.php and launch.php. 

It's also giving me a 400 error for one activity that was previously set to version 0.9. (It previously also gave me a 400 error because it was configured to 0.9, but this fix should resolve that). 

I think this needs a good test before I review again.

File: lib.php (hunk 1 above)
Comment (davidpesce) [inline lib.php:152]: This is actually from global/activity settings change.

**Comment by davidpesce** [comment]
I'm going to close this PR because it's still got a lot of the commits from the global settings. I'll submit a new one with just the removeversion branch.
